apps/site/src/views/Landing/Landing.tsx

```diff
@@ -0,0 +1,9 @@
+import "./Landing.module.scss";
```

Comment:
Pretty sure we don't need the `src/views/Landing/` folder because the Landing component that's being used is in `src/views/Home/sections`.

Reply:
fixed